bayesian/api_v1.py

```diff
@@ -704,7 +704,7 @@ def post():
                 filepath = filepaths[index]
                 content = manifest_file_raw.read().decode('utf-8')
             # For npm-list.json, we need not verify it as its not going to mercator task
-            if origin != "vscode" and ecosystem not in ("npm", "pypi", "golang"):
+            if origin != "vscode" and resolved_files_exist(filename) is False:
```

Comment:
the function `resolved_files_exist(filename)` returns boolean in all cases, so you can use `not resolved_files_exist(filename)`

Reply:
Something like 
`if origin != "vscode" and not resolved_files_exist(filename):`

Reply:
done